integtest/pom.xml

```diff
@@ -19,7 +19,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <hadoop.version>2.5.2</hadoop.version>
-        <spark.version>2.1.0</spark.version>
+        <spark.version>2.1.1</spark.version>
```

Comment:
Setting spark version to the same as that in TiSpark pom file to make IDE resolve one consistent version of spark instead of two.